### ISSUE TITLE
external audio playback is broken

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#34027]
See 
![image](https://user-images.githubusercontent.com/42591237/189645470-4ad4fd1d-5f47-48ab-b0ce-9cfab0d60b11.png)

**Comment by max-nextcloud** [comment]
Actually we are not using the viewer for audio mimetypes yet.

Tried locally and `mp3` file had hasPreview and then tried to load a preview image for the audio file.
Enabling the viewer for audio files worked pretty well. However the layout does not really work:

![Bildschirmfoto vom 2022-09-13 13-54-06](https://user-images.githubusercontent.com/97337118/189903420-7d95c7c2-7a5b-47ab-9920-728180b2f7af.png)

In particular the settings menu will be totally hidden by the top menu

**Comment by szaimen** [comment]
Another option would be merging my PR and creating a new PR for this afterwards. WDYT?

**Comment by max-nextcloud** [comment]
> Another option would be merging my PR and creating a new PR for this afterwards. WDYT?

Yes... let's go step by step. These seem to be independent problems.

**Comment by max-nextcloud** [comment]
I'll open another PR with my change.

**Comment by szaimen** [comment]
Can you maybe review my PR then? This should unblock this one...

**Comment by szaimen** [comment]
> I'll open another PR with my change.

okay, please note that you will likely run into conflicts though if you open it before my change is merged...

**Comment by max-nextcloud** [comment]
> > I'll open another PR with my change.
> 
> okay, please note that you will likely run into conflicts though if you open it before my change is merged...

https://github.com/nextcloud/server/pull/34051/files

For now it's a single line change that is independent of what you are doing.

**Comment by szaimen** [comment]
> For now it's a single line change that is independent of what you are doing.

okay. After the other PR is merged we still need to rebase in order to be able to adjust the styling...
